webapp/src/ts/app.component.ts

```diff
@@ -159,6 +156,13 @@ export class AppComponent implements OnInit, AfterViewInit {
       });
   }
 
+  private registerMaterialIcons() {
```

Comment:
> This PR changes "X" icon in filter and FAB to fit with Figma design - before we didn't have time to do it, so doing it now since the new menu needs it too.

The filter, fab and menu have the same X now. The MatIcon requires registering the SVG images, that's how Angular Material defined it.

Reply:
Before we were using Fontawesome, which is also registered in MatIcon, but the X is bolder and has a different shape than the Figma design.

Reply:
Thanks